steps/dropbox-upload-v2/0.0.1/step.yml

```diff
@@ -0,0 +1,62 @@
+title: dropbox-upload-v2
+summary: |
+  upload a file to dropbox and get a shareable link in return
+description: |
+  using this step you can send downloadable links via any platform
+website: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2
+source_code_url: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2
+support_url: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2/issues
+published_at: 2021-08-23T11:22:49.406953+03:00
+source:
+  git: https://github.com/AlexPinhasov/bitrise-step-dropbox-upload-v2.git
+  commit: 328cf9a232efcf656bd62fdf3bfd692c733b7f88
```

Comment:
This commit should reside in a default branch.
Now you have empty default `master` and actual commits only on `main` which makes harder to see the actual source code using bitrise UI.
You should change the default branch in your repo.

Reply:
Done